docs/quickstarts/insights-tasks-conversion/insights-tasks-conversion.yml

```diff
@@ -1,6 +1,6 @@
 # Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
 metadata:
-  name: insights-tasks-convert
+  name: insights-tasks-conversion
```

Comment:
Can we double-check if the names of pre-conversion, conversion and LEAPP quick-starts are the same as outlined in https://issues.redhat.com/browse/RHINENG-7706?focusedId=24329052&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24329052? I am a bit confused why we are renaming it here to "insights-tasks-conversion" instead of "conversion-quickstart"